examples/inference-deployments/mpt/mpt_7b_ft_handler.py

```diff
@@ -0,0 +1,232 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import configparser
+import copy
+import os
+from typing import Any, Dict
+
+import torch
+from scripts.inference.convert_hf_mpt_to_ft import convert_mpt_to_ft
+from torch.nn.utils.rnn import pad_sequence
+from transformers import (AutoConfig, AutoModelForCausalLM, AutoTokenizer,
+                          pipeline)
+
+from examples.pytorch.gpt.utils.parallel_gpt import ParallelGPT
+
+LOCAL_CHECKPOINT_PATH = '/tmp/mpt'
+
+
+class MPTFTModelHandler:
+
+    DEFAULT_GENERATE_KWARGS = {
```

Comment:
more generally, are these kwargs documented anywhere? it seems like they might be slightly different from normal HF kwargs?

Reply:
These are not very well documented. So I have added one line explanation for the ones I could find.